tests/constants.rs

```diff
@@ -0,0 +1,5 @@
+#[allow(dead_code)]
+pub const UNINITIALIZED_BYTE: u8 = 0xDE;
```

Comment:
Probably you wanted `#[cfg(test)]` but if/when you put them in the `tests` module this will be a non-issue because they will be accessible outside of *ring* can thus never dead.

Let's use `U8` instead of `BYTE` in the name.